scripts/cargo-install-all.sh

```diff
@@ -201,12 +201,7 @@ check_dcou() {
     # shellcheck source=scripts/spl-token-cli-version.sh
     source "$SOLANA_ROOT"/scripts/spl-token-cli-version.sh
 
-    # the patch-related configs are needed for rust 1.69+ on Windows; see Cargo.toml
-    # shellcheck disable=SC2086 # Don't want to double quote $rust_version
-    "$cargo" $maybeRustVersion \
-      --config 'patch.crates-io.ntapi.git="https://github.com/solana-labs/ntapi"' \
-      --config 'patch.crates-io.ntapi.rev="97ede981a1777883ff86d142b75024b023f04fad"' \
-      install --locked spl-token-cli --root "$installDir" $maybeSplTokenCliVersionArg
+    "$cargo" $maybeRustVersion install --locked spl-token-cli --root "$installDir" $maybeSplTokenCliVersionArg
```

Comment:
nit: seems the needed `shellcheck disable=...` is missing...

Reply:
yeah... fixed [6bb361c](https://github.com/anza-xyz/agave/pull/3584/commits/6bb361c530e8bb4c9668a8467a5cd6b1c0809462)